docs/getting-started.md

```diff
@@ -0,0 +1,94 @@
+# Getting Started
+
```

Comment:
Maybe emphasize more that this is only instructing how to use a **local** topology.
For example insert:

`## Connecting to a Locally Running Test Network on Your Machine`

We already had the problem with the public docs that people often got confused about the different networks (ScionLab vs Local Network (locally running test network) vs Local AS (part of production network), ....).

Also, maybe add some hints to instruction how to connect to the normal network (local AS)?

Reply:
I've made some changes to hopefully clarify this.

Regarding the production network. We don't have a bootstrapping mechanism yet, so I guess it's anyways not really possible to connect to it without knowing the `topology.json`.

Reply:
I think bootstrapping may be quite important, it would be good if we had that. We probably don't need all available bootstrapping features, but a subset would be nice. @marcfrei ?

Reply:
Yes, I agree. We should add bootstrapping (at least in a simple form) to the TODO list.

Reply:
I've created a ticket #43